### PR TITLE
Created Signal Bar component

### PR DESCRIPTION
**Pull request by disneyonbryce**
If you actually run the code, you will find the .svg pictures to be very low quality. I propose to replace them with the same pictures but with a higher resolution/size.

Additionally, I will need help with importing the Oswald font. @Hesham-E

### CODE REVIEW

File: src/SignalBar.qml [hunk 1]
```diff
@@ -0,0 +1,197 @@
+import QtQuick 2.15
+
+Item {
+    id: container
+    //Consider enums/constants for properties?
```
Comment (disneyonbryce) [inline src/SignalBar.qml:5]: Asked in Discord - 2021-07-10

**Comment by DatLam25** [comment]
I'm getting errors when I'm running the project. 
Screenshot from 2021-07-15 19-47-21

**Comment by DatLam25** [comment]
For some reason, your SignalBar.qml isn't included in the qml.qrc either.

**Review by JessieG123 — CHANGES_REQUESTED** [review]
Remember to merge master in to include travis.yml before merging!

**Review by JessieG123 — APPROVED** [review]
Checked the git commit history and I think it should be ok.  LGTM on this the git things :D 

Make sure to rebase your commits into one commit with the commit message following the format I mentioned on discord when you're ready to merge. 

```
SFT-### Title

Description of changes
```

**Pull request closed without merging**